website/blog/2018-05-23-1.13.0.md

```diff
@@ -551,58 +551,132 @@ When using `--prose-wrap preserve`, Prettier will no longer merge continuous CJK
-
-#### feat: use `postcss-values-parser` for `@namespace` at rule ([#4373](https://github.com/prettier/prettier/pull/4373) by[@evilebottnawi](https://github.com/evilebottnawi))
-
-#### feat: improve `@support` at-rule output ([#4372](https://github.com/prettier/prettier/pull/4372) by [@evilebottnawi](https://github.com/evilebottnawi))
-
-#### feat(typescript): support TaggedTemplateExpression typeParameters ([#4353](https://github.com/prettier/prettier/pull/4353) by [@ikatyang](https://github.com/ikatyang))
-
-#### feat: adding plugin ability for isBlockComment ([#4347](https://github.com/prettier/prettier/pull/4347) by [@mgrip](https://github.com/mgrip))
-
-#### fix(glimmer): preserve leading/trailing spaces in text nodes undernea… ([#4288](https://github.com/prettier/prettier/pull/4288) by [@n8n8baby](https://github.com/n8n8baby))
-
-#### feat(typescript): support generic JSX element ([#4268](https://github.com/prettier/prettier/pull/4268) by [@ikatyang](https://github.com/ikatyang))
-
-#### Allow trailing comma in scss list and maps ([#4317](https://github.com/prettier/prettier/pull/4317) by [@ad1992](https://github.com/ad1992))
-
-#### Improve formatting of TS casts with generics and unions ([#4219](https://github.com/prettier/prettier/pull/4219) by [@kpdonn](https://github.com/kpdonn))
+#### Fix: Always preserve frontmatter, remove dependency ([#4392] by [@eliasmeire])
+
+#### Fix: Group long name method in bind expressions ([#4447] by [@misoguy])
+
+#### Fix plugin API in globally installed Prettier and introduce optional --plugin-search-dir ([#4192] by [@kachkaev])
+
+#### Fix: Correctly break class property initializers ([#4442] by [@nicolo-ribaudo])
+
+#### feat(typescript): support TSImportType ([#4429] and [#4438] by [@ikatyang])
+
+#### Fix TSMappedType comments ([#4418] by [@duailibe])
+
+#### Fix cursor offset tracking ([#4397] by [@ds300])
+
+#### Fix trailing JSDoc comments with trailing regular comments ([#4414] by[@duailibe])
+
+#### Prevent flattening multiplicative operations ([#4407] by [@duailibe])
+
+#### fix spacing issue in postcss simple vars ([#4408] by [@ad1992])
+
+#### Wrap modulo operation in parens inside additive operation ([#4413] by [@duailibe])
+
+#### feat: use `postcss-values-parser` for `@namespace` at rule ([#4373] by[@evilebottnawi])
+
+#### feat: improve `@support` at-rule output ([#4372] by [@evilebottnawi])
+
+#### feat(typescript): support TaggedTemplateExpression typeParameters ([#4353] by [@ikatyang])
+
+#### feat: adding plugin ability for isBlockComment ([#4347] by [@mgrip])
+
+#### fix(glimmer): preserve leading/trailing spaces in text nodes undernea… ([#4288] by [@n8n8baby])
+
+#### feat(typescript): support generic JSX element ([#4268] by [@ikatyang])
+
+#### Allow trailing comma in scss list and maps ([#4317] by [@ad1992])
+
+#### Improve formatting of TS casts with generics and unions ([#4219] by [@kpdonn])
+
+[@1wheel]: https://github.com/1wheel
```

Comment:
FYI, this can cause some issues if a link appears above the fold. But it's not a problem yet in this file.

Reply:
Also, did you use a tool to automate this? If so, I'd like to know what it is so I can add it to [prs-merged-since](https://www.npmjs.com/package/prs-merged-since)

Reply:
@ikatyang ^

Reply:
Just a simple script 😅

```js
"use strict";

const fs = require("fs");

const filename = "2018-05-23-1.13.0.md";
const text = fs.readFileSync(filename, "utf8");

const links = [];
const extracted = text.replace(
  /\[(@[^\]]+|#\d+)\]\(([^)]+)\)/g,
  (_, title, link) => {
    links.push(`[${title}]: ${link}`);
    return `[${title}]`;
  }
);

console.log(
  extracted +
    "\n" +
    Array.from(new Set(links))
      .sort()
      .join("\n")
);
```

Reply:
If it works, it works :) thanks for sharing

Reply:
@suchipi I used prs-merged-since BTW it was great! I'm gonna open a PR to extract links later :)